src/script/aprg/README.rst

```diff
@@ -0,0 +1,61 @@
+# aprg - Automatic Performance Report Generator - POC toolkit
+
+This is a POC/prototype of a number of ideas to:
+
+- run mechanically a set of workloads using FIO with RBD
+- get some measurements for CPU utilisation, memory
```

Comment:
gather measurements for CPU utilization and RAM

Reply:
Thank you @anthonyeleven for your comments, I've updated the PR 👍